loki/frontend/omni.py

```diff
@@ -215,6 +215,9 @@ def visit_FuseDecl(self, o, **kwargs):
         if module is not None:
             # Import symbol attributes from module, if available
             for k, v in module.symbol_attrs.items():
+                # Don't import private module symbols
+                if v.private is True or (module.default_access_spec == "private" and v.public is None):
```

Comment:
A suggestion to make the expression a little shorter and more pythonic:
```suggestion
                if v.private or (module.default_access_spec == "private" and not v.public):
```